DQM/HLTEvF/plugins/LumiMonitor.cc

```diff
@@ -0,0 +1,212 @@
+#include "DQM/HLTEvF/plugins/LumiMonitor.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+
+
+// -----------------------------
+//  constructors and destructor
+// -----------------------------
+
+LumiMonitor::LumiMonitor( const edm::ParameterSet& iConfig ) : 
+  conf_(iConfig) 
+  , doPixelLumi_                  ( conf_.getParameter<bool>  ("doPixelLumi")                 )
+  , useBPixLayer1_                ( conf_.getParameter<bool>  ("useBPixLayer1")               )
+  , minNumberOfPixelsPerCluster_  ( conf_.getParameter<int>   ("minNumberOfPixelsPerCluster") )
+  , minPixelClusterCharge_        ( conf_.getParameter<double>("minPixelClusterCharge")       )
+  
+{
+
+  folderName_ = conf_.getParameter<std::string>("FolderName");
```

Comment:
can you move `folderName_`, `pixelClustersToken_` and `lumiscalersToken_` to the initialisation as well ?